integrations/docker/images/stage-3/chip-build-tizen-qemu/Dockerfile

```diff
@@ -147,17 +150,18 @@ RUN set -x \
     "RemainAfterExit=yes\n" \
     | guestfish --rw -a $TIZEN_IOT_IMAGE_ROOT -m /dev/sda upload - $SYSTEMD_UNIT_OPT_POST_MOUNT : \
     ln-sf $SYSTEMD_UNIT_OPT_POST_MOUNT $SYSTEMD_REQUIRES_LOCAL_FS \
-    # Mount Matter ISO image on startup
-    && SYSTEMD_UNIT_CHIP_MOUNT=$SYSTEMD_SYSTEM/mnt-chip.mount \
+    # Mount 9P at /mnt/chip
+    && SYSTEMD_9P_MOUNT=$SYSTEMD_SYSTEM/mnt-chip.mount \
     && echo -n \
     "[Unit]\n" \
-    "ConditionPathIsMountPoint=!/mnt/chip\n" \
+    "After=network.target\n" \
     "[Mount]\n" \
-    "What=/dev/disk/by-label/CHIP\n" \
+    "What=host0\n" \
```

Comment:
we have to change version for dockerfiles for any dockerfile changes.

Are the tizen qemu setup something that can be pulled into the vscode image so that we can test in the "one large image for everyting" environment?

Reply:
This PR is now for the Dockerfile part only.

Reply:
Updated vscode container's env vars.

Reply:
> Are the tizen qemu setup something that can be pulled into the vscode image so that we can test in the "one large image for everyting" environment?

Yes. If the vscode image will be "based" on the chip-build-tizen-qemu docker image instead of chip-build-tizen, it will be possible to build and run tests in vscode.